test/test_dispatch.py

```diff
@@ -181,15 +188,16 @@ def go(ctor_order):
             for ctor_order in itertools.permutations(range(len(ops))):
                 go(ctor_order)
 
-        # Return the "full" state after all operations are run.
+        # Return the "full" state and computed dispatch table after
+        # all operations are run.
         # If this KeyErrors, that means that there did not exist any
         # ordering of ctors which got us to the "end".  That's an
         # error in test construction: it means you could have
         # factored the test into two smaller ones.
-        return results[frozenset(range(len(ops)))][0]
+        return results[frozenset(range(len(ops)))][:-1]
```

Comment:
It's not obvious to me that you get a 2-size tuple here. Also, consider returning a named tuple, so that you can avoid ugly `[0]` indexing.